README.md

```diff
@@ -27,13 +27,13 @@ To build this project, you must use Git to clone the repository, instead of down
 git submodule update --init --recursive
 ```
 
-Building the solution for **any** platform requires Visual Studio 2022 17.8 or newer and/or the .NET SDK 8.0.200. A modern version of Visual Studio Code, MonoDevelop or Visual Studio for Mac could also work, but are not officially supported.
-To debug WindowsXNA builds the .NET SDK 8.0 x86 is additionally required.
+Building the solution for **any** platform requires the .NET SDK 10.0.100. Editing the source code requries Visual Studio 2026 or newer. A modern version of Visual Studio Code, MonoDevelop or Rider could also work, but are not officially supported.
```

Comment:
Rider *is* officially supported (source: I use it)

Reply:
> Building the solution for **any** platform requires the .NET SDK 10.0.100. Editing the source code requries Visual Studio 2026 or newer, or Rider 2025.3 or newer. A modern version of Visual Studio Code also works, but is not officially supported.